.github/workflows/release.yml

```diff
@@ -0,0 +1,61 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseVersion:
+        type: string
+        required: true
+        description: The version of this release. Must be a semantic version of the form X.Y.Z.
+      dry_run:
+        type: boolean
+        required: true
+        default: false
+        description: Dry run, will not push branches or upload the artifacts.
+      pre_release:
```

Comment:
For now, this needs to always be `true` or we will accidentally replace the "production" release on npm with the 1.x release. We can decide when we want to do that, but for now at least default to true, and possibly force it to true.

Reply:
Good point. I think the condition I've added will always publish with the `--next` flag and never without the `--next` flag.

Reply:
I'd be a lot more comfortable if we just removed this for now. Is there a reason to leave it in?

Reply:
Good point, I've tried to make the changes as small as possible but you are very right, removing the risk completely is way better.
I've changed it to remove the input and all references/actions we do not want to tigger.